packages/neuron-wallet/src/block-sync-renderer/sync/queue.ts

```diff
@@ -55,13 +55,45 @@ export default class Queue {
       }
     })
 
-    while (!this.stopped) {
+    // while (!this.stopped) {
+    //   try {
+    //     this.inProcess = true
+
+    //     if (this.lockHashes.length !== 0) {
+    //       const startNumber = this.currentBlockNumber
+    //       const endNumber = this.currentBlockNumber + BigInt(this.fetchSize)
+    //       const realEndNumber: bigint = endNumber < this.endBlockNumber ? endNumber : this.endBlockNumber
+
+    //       if (realEndNumber >= this.endBlockNumber) {
+    //         this.yieldTime = 1000
+    //       } else {
+    //         this.yieldTime = 1
+    //       }
+
+    //       if (realEndNumber >= startNumber) {
+    //         const rangeArr = ArrayUtils.rangeForBigInt(startNumber, realEndNumber).map(num => num.toString())
+    //         await this.pipeline(rangeArr)
+    //       }
+    //     }
+    //   } catch (err) {
+    //     if (err.message.startsWith('connect ECONNREFUSED')) {
+    //       logger.debug(`Sync:\terror:`, err)
+    //     } else {
+    //       logger.error(`Sync:\terror:`, err)
+    //     }
+    //   } finally {
+    //     await CommonUtils.sleep(this.yieldTime)
+    //     this.inProcess = false
+    //   }
+    // }
+    setTimeout(async()=>{
```

Comment:
?

Reply:
> ?

My fault, it's a workaround for the sync problem locally.

Reply:
It's been removed